go.mod

```diff
@@ -5,6 +5,7 @@ go 1.21
 require (
 	cosmossdk.io/api v0.7.2
 	cosmossdk.io/client/v2 v2.0.0-beta.1
+	cosmossdk.io/collections v0.4.0
```

Comment:
Are these changes not part of the plugin work?

Reply:
amazing catch!!